index2.html

```diff
@@ -102,16 +102,14 @@
                               </div>
                               <div class="col-lg-4 col-md-12 clearfix">
```

Comment:
@stefanispayde these two `div`s look empty to me, are they left for a reason? 
my assumption is this is where the images were, so you probably want to remove them completely.

Reply:
Removed empty div tags for unused deleted images